src/core/bpf-devices.c

```diff
@@ -276,14 +276,6 @@ int bpf_devices_supported(void) {
                 return supported = 0;
         }
 
-        r = cg_unified_controller(SYSTEMD_CGROUP_CONTROLLER);
```

Comment:
The whole function can be dropped really with our baseline being 5.4 (except for the geteuid check). Maybe add a TODO above?

Reply:
Done.